lib/compliance/parsepdf417-us.js

```diff
@@ -20,6 +20,15 @@ var CODES = {
   DAQ: 'documentNumber'
 }
 
+const states = {
+  'CA': ['AB', 'BC', 'MB', 'NB', 'NL', 'NT', 'NS', 'NU', 'ON', 'PE', 'QC', 'SK', 'YT'],
+  'USA': ['AL', 'AK', 'AZ', 'AR', 'CA', 'CO', 'CT', 'DE', 'FL', 'GA',
+    'HI', 'ID', 'IL', 'IN', 'IA', 'KS', 'KY', 'LA', 'ME', 'MD',
+    'MA', 'MI', 'MN', 'MS', 'MO', 'MT', 'NE', 'NV', 'NH', 'NJ',
+    'NM', 'NY', 'NC', 'ND', 'OH', 'OK', 'OR', 'PA', 'RI', 'SC',
+    'SD', 'TN', 'TX', 'UT', 'VT', 'VA', 'WA', 'WV', 'WI', 'WY']
+}
+
```

Comment:
Two letter abbreviations for U.S Territories are missing, and also the letter abbreviation for District of Columbia (DC)
We should ask if these abbreviations should also be included or not
https://www.ncbi.nlm.nih.gov/books/NBK7254/

Reply:
I added them, just in case. Shouldn't be much of a problem either way.